### ISSUE TITLE
the same data, the same code , different thread number, different result, why ?

### ISSUE THREAD

**Issue by yale8848** [xiamx/fastText#1]
the same data, the same code , different thread number, different result, why ?

fastText.exe  supervised -input train.dat -output model -epoch 25  -loss softmax -ws 5 -thread 10
fastText.exe test model.bin test.dat
P@1  	0.874
R@1	0.874

fastText.exe supervised -input train.dat -output model -epoch 25  -loss softmax -ws 5 -thread 20
fastText.exe test model.bin test.dat
P@1	        0.939
R@1	0.939

**Comment by creat89** [comment]
Have you tried to run again the models? Remember that the weights of neural networks at the beginning are random. This could be the reason.
